utils/notification_service.py

```diff
@@ -1225,6 +1226,20 @@ def pop_default(l: list[Any], i: int, default: Any) -> Any:
 
                 stacktraces = handle_stacktraces(artifact["failures_line"])
 
+                # Add the captured actual outputs for patched methods (`torch.testing.assert_close`, `assertEqual` etc.)
+                if "captured_info" in artifact:
+                    step_number = None
+                    for step in job.get("steps", []):
+                        if step["name"] == "Captured information":
+                            step_number = step["number"]
```

Comment:
Could this crash if somehow the step has no `name` or `number` ? Not sure this can happen but maybe a `.get` would be safer

Reply:
Mb not familiar enough with the API :)